src/connectors/breadcrumb/connectBreadcrumb.js

```diff
@@ -81,11 +81,11 @@ export default function connectBreadcrumb(renderFn, unmountFn = noop) {
               'Using Breadcrumb and HierarchicalMenu on the same facet with different options overrides the configuration of the HierarchicalMenu.'
             );
 
-            return {};
+            return currentConfiguration;
           }
         }
 
-        return {
+        return currentConfiguration.setQueryParameters({
```

Comment:
No need to add `hierarchicalFacetsRefinements`?

Reply:
good point, this is missing. I originally didn't add it because someone usually uses it with a hierarchical or a configure which sets that, but it's still better to avoid the merging issues completely

Reply:
done in c195b49ed